datafusion/functions/src/datetime/from_unixtime.rs

```diff
@@ -93,12 +124,59 @@ fn get_from_unixtime_doc() -> &'static Documentation {
         Documentation::builder()
             .with_doc_section(DOC_SECTION_DATETIME)
             .with_description("Converts an integer to RFC3339 timestamp format (`YYYY-MM-DDT00:00:00.000000000Z`). Integers and unsigned integers are interpreted as nanoseconds since the unix epoch (`1970-01-01T00:00:00Z`) return the corresponding timestamp.")
-            .with_syntax_example("from_unixtime(expression)")
+            .with_syntax_example("from_unixtime(expression, timezone)")
             .with_argument(
                 "expression",
-                "Expression to operate on. Can be a constant, column, or function, and any combination of arithmetic operators."
+                "Expression to operate on. Can be a constant, column, or function, and any combination of arithmetic operators.",
             )
+            .with_argument(
+                "timezone",
+                "Optional timezone to use when converting the integer to a timestamp. If not provided, the default timezone is UTC.",
+            )
+
             .build()
             .unwrap()
     })
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
These tests are great to have but it would be very nice to have a few additional ones in timestamps.slt as well

Reply:
added